src/pudl/transform/ferc1.py

```diff
@@ -3848,15 +3994,6 @@ def process_xbrl_metadata(self, xbrl_metadata_json) -> pd.DataFrame:
         meta = (
             super()
             .process_xbrl_metadata(xbrl_metadata_json)
-            .assign(
```

Comment:
To confirm, these are all now reflected in the `rename_columns_ferc1` parameters? If so, they aren't being added in this PR so I'd just like to confirm!

Reply:
sorry i should have done this separately.. these little deletions in the bespoke-table `process_xbrl_metadata` should have happened in the base branch once i integrated the `rename_column_xbrl_to_pudl` into the standard `process_xbrl_metadata`